template/pages/index.vue

```diff
@@ -18,14 +18,12 @@
 
             <ul class="hoz sml-push-y2 med-push-y3">
               <li>
-                <a href="#TODO-section-1"
-                   @click.prevent="scrollTo('#TODO-section-1')">
-                  Link to section 1
+                <a @click.prevent="scrollTo('#sign')">
```

Comment:
This will likely be the large CTA button, and it's fairly straightforward to switch it around in a new project.